scripts/lint.sh

```diff
@@ -15,19 +15,31 @@ fi
 
 cd example/
 flutter pub get
-result=`dartanalyzer .`
+result=$(dartanalyzer .)
```

Comment:
you could use a similar analysis to check this example as well (not sure if worth the effort)